release/models/keychain/openconfig-keychain-types.yang

```diff
@@ -35,12 +41,6 @@ module openconfig-keychain-types {
     "Base identify to define the type of authentication";
   }
 
-  identity NONE {
-    base AUTH_TYPE;
-    description
-    "NO authentication is used";
-  }
-
```

Comment:
I understand the desire to remove this leaf, but considering backwards compatibility, is it necessary to remove this identity?

Reply:
Today there is no leaf using this type 'AUTH_TYPE' so I think there is no backward compatibility problem.

Reply:
I think that this is unreferenced, as you say, so I'm OK with removing it.

Reply:
I was thinking of the use case where config generation software in the community has been written to utilize this leaf would be affected by its removal.  FWIW, I scanned our code and we are not using this identity, so from a selfish point of view, I am ok with removing it as well. Marking resolved.